profile_bluesky/startup/instrument/devices/scaler.py

```diff
@@ -48,20 +121,20 @@ def select_plot_channels(self,chan_names):
 
         for ch in name_map.keys():
             try:
-                if ch in chan_names: 
-                    getattr(self.channels,name_map[ch]).kind = Kind.hinted
+                if ch in chan_names:
+                    getattr(self.channels, name_map[ch]).kind = Kind.hinted
                 else:
-                    if getattr(self.channels,name_map[ch]).kind.value != 0:
-                        getattr(self.channels,name_map[ch]).kind = Kind.normal
+                    if getattr(self.channels, name_map[ch]).kind.value != 0:
+                        getattr(self.channels, name_map[ch]).kind = Kind.normal
                     else:
-                        getattr(self.channels,name_map[ch]).kind = Kind.omitted
+                        getattr(self.channels, name_map[ch]).kind = Kind.omitted
             except KeyError:
                 raise RuntimeError("The channel {} is not configured "
                                    "on the scaler.  The named channels are "
                                    "{}".format(ch, tuple(name_map)))
-                
+
     def select_channels(self, chan_names):
```

Comment:
In [ophyd](https://github.com/bluesky/ophyd/pull/887), `chan_names=None` is now defined so it is not necessary to provide `None`